fastlane.gemspec

```diff
@@ -35,6 +35,9 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'credentials_manager', '>= 0.11.0', '< 1.0.0' # Password Manager
   spec.add_dependency 'spaceship', '>= 0.14.2', '< 1.0.0' # communication layer with Apple's web services
 
+  spec.add_dependency 'appium_lib', '>= 4.1.0' # UI testing with Appium
```

Comment:
We can't add new dependencies just for new actions. Instead take a look at the `cocoapods.rb` action: basically you run `verify_gem!("name")` to make sure a certain gem is installed. The user has to add the gem to their `Gemfile`